packages/@react-spectrum/dialog/test/DialogContainer.test.js

```diff
@@ -175,7 +166,45 @@ describe('DialogContainer', function () {
 
     triggerPress(button);
     act(() => {jest.runAllTimers();});
+    act(() => {jest.runAllTimers();});
 
     expect(queryByRole('dialog')).toBeNull();
   });
+
+  it('should be able to have dialogs open dialogs and still restore focus', function () {
```

Comment:
```suggestion
  it('should be able to have menus open dialogs and still restore focus', function () {
```

Reply:
this is correct, the menu opens a dialog but the dialog opens another dialog